runtime/lua/vim/ui/clipboard/osc52.lua

```diff
@@ -9,10 +9,34 @@ local function osc52(clipboard, contents)
   return string.format('\027]52;%s;%s\027\\', clipboard, contents)
 end
 
+---@class clipboard_cache
+---
+---@field lines_str string String representation of lines passed to the copy() funciton
+---@field regtype string regtype argument passed to the copy() function
+---
+---@type table<string, clipboard_cache>
+M.cache = {}
```

Comment:
This can be a local. Doesn't need to be exposed publicly.

```suggestion
local cache = {}
```

Reply:
Implemented in https://github.com/neovim/neovim/pull/34228/commits/1b89652e78afed3876bdbbc7a74bc16536929fec